samples/listctrl/listtest.cpp

```diff
@@ -377,6 +379,18 @@ void MyFrame::OnToggleBell(wxCommandEvent& event)
     m_listCtrl->EnableBellOnNoMatch(event.IsChecked());
 }
 
+void MyFrame::OnCheckVisibility(wxCommandEvent& WXUNUSED(event))
+{
+    if( m_listCtrl->IsVisible( 2 ) )
```

Comment:
Maybe check visibility of the selected line(s) instead of fixed 2 and 9?

Reply:
@MaartenBent ,
I don't know about this one?
I'm just checking one at the top and one at the bottom, making sure it will work in different scenarios.